mbq/atomiq/utils.py

```diff
@@ -27,3 +31,54 @@ def wrapped_func(*args, **kwargs):
 
         return wrapped_func
     return wrapper
+
+
+def has_user_transactions_in_django_test_case():
+    """
+    Atomiq publish will throw an exception if not used within a transaction, and we want
+    to make sure nobody ships code that is going to throw this exception in production.
+    The problem is that Django TestCase wraps all test functions in a transaction, which
+    masks this error. Therefore, we use this function to ~inspect~ ~the~ ~stack~ and look for
+    instances of django.test.TestCase. If we find TestCase unit test, then we expect there to be
+    TWO transactions created by Django, and ONE transaction created by the user.
```

Comment:
Solid docstring but you need to update it to reflect the difference between setup and being in the test methods